kolibri/core/urls.py

```diff
@@ -46,14 +49,24 @@
 
 app_name = 'kolibri'
 
-
-urlpatterns = plugin_urls()
-
-urlpatterns += static(paths.get_content_url("/"), document_root=paths.get_content_dir_path())
-
-urlpatterns += [
-    url(r'^i18n/setlang/$', set_language, name='set_language'),
+core_urlpatterns = [
     url(r'^$', RootURLRedirectView.as_view()),
-    url(r'^guestaccess/', GuestRedirectView.as_view(), name="guest"),
+    url(r'^i18n/setlang/$', set_language, name='set_language'),
+    url(r'^redirectuser/$', RootURLRedirectView.as_view(), name="redirect_user"),
+    url(r'^guestaccess/$', GuestRedirectView.as_view(), name="guest"),
     url(r'^logout/$', logout_view, name='logout'),
+    url(r'^api/', include('kolibri.core.api_urls')),
+    url(r'', include('kolibri.core.content.urls')),
 ]
+
+
+urlpatterns = [
+    url(r'^admin/', include(admin.site.urls)),
```

Comment:
I'm wondering if this should be refactored into `kolibri.deployments.default.urls` ? Same for `morango`. Mounting other applications would typically happen at project-level, not inside an application.

That way, we could define `core` as a default application namespace rather than an instance namespace: https://docs.djangoproject.com/en/1.11/topics/http/urls/#url-namespaces

In the longer run, it will make Kolibri more flexible, in case it is remixed with other types of Django applications, or a project for instance wants `/kolibri` and `/admin`.